indexing-service/src/main/java/io/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -182,14 +189,17 @@ public Response getWorkerConfig()
     return Response.ok(workerConfigRef.get()).build();
   }
 
+  // default value is used for backwards compatibility
   @POST
   @Path("/worker")
   @Consumes(MediaType.APPLICATION_JSON)
   public Response setWorkerConfig(
-      final WorkerBehaviorConfig workerBehaviorConfig
+      final WorkerBehaviorConfig workerBehaviorConfig,
+      @HeaderParam("author") @DefaultValue("") final String author,
```

Comment:
Since those are not standard HTTP headers, we should prefix them with X-Druid, in line with what we do for the query response context.